docs/source/user_manual/annotated_examples.rst

```diff
@@ -450,17 +450,6 @@ source: `tabbed_plots.py <https://github.com/enthought/chaco/tree/master/example
 .. image:: example_images/tabbed_plots1.png
 .. image:: example_images/tabbed_plots2.png
 
-``traits_editor.py``
```

Comment:
I guess the information in this example is duplicated in other examples so no problem getting rid of it.